### ISSUE TITLE
[QA] Clipping issues with some wearables 

### ISSUE THREAD

**Issue by DafGreco** [decentraland/unity-explorer#2300]
### **Explorer Alpha build versions:**
0.29.0

### **Issue Description:**
There are some wearables that cannot be completely seen by the user when using them 

### **STR:** 

  1. Open 0.29.0
  2. Go to the backpack
  3. Change outfits 
  4. Validate that the outfit cannot be completely seen by the user 

### **Expected behavior:**
The clothes should be able to be seen properly by the user when changing them on the backpack

### **Current behavior:**
There are some wearables that cannot be completely seen by the user when using them 

#### **Evidence:**
![image (5)](https://github.com/user-attachments/assets/651a2afa-39df-47c7-a211-23b012ba554b)
![image (6)](https://github.com/user-attachments/assets/7e599a67-5889-480d-8959-dd519567bab4)
![image (4)](https://github.com/user-attachments/assets/01302258-57c3-47a2-accd-74beb1cd7de9)

#### **Additional Notes:**
Happens on Windows 11 PRO
16 GB RAM

**Comment by anicalbano** [comment]
This might be related, but the following wearable: https://decentraland.org/marketplace/contracts/0x642615cc0f4abd6487be1e03e6fd69adbf7f553f/items/0 also has an issue with the view. Seems like in world there is only one wing visible but if you look at the wearable from one side, you can see both of them:

![image](https://github.com/user-attachments/assets/67989809-3bb5-4283-a32f-d53abdb4fed2)
![image](https://github.com/user-attachments/assets/af341724-b082-4f09-8f1f-16c598483a18)

**Comment by ShibuyaMartin** [comment]
can we have the glbs to check them? it seems is due to the backface culling

**Comment by ShibuyaMartin** [comment]
if you point me the links on the marketplace i can also download them from the servers @DafGreco

**Comment by Kaze-No-Kai** [comment]
Another issue most likely related to clipping: https://decentraland.org/marketplace/contracts/0xa475aa972fc7b5164f80ea4d5534313f4a584230/items/0
![Screenshot_2024-10-29_053232](https://github.com/user-attachments/assets/0c5f67ed-bd86-4bfe-b96a-68ad59b56e6e)

**Comment by Kaze-No-Kai** [comment]
Another example of a wearable with a possible clipping issue:
https://decentraland.org/marketplace/contracts/0x222deaa90399023e707abd3f81b268493bdc891a/items/2
![Uploading (Stevame) cigar wearable doesn't work.png…]()

**Comment by Kaze-No-Kai** [comment]
hey @ShibuyaMartin regarding the 1 side render, is this related to this wearable?
https://decentraland.org/marketplace/contracts/0x642615cc0f4abd6487be1e03e6fd69adbf7f553f/items/0
I am asking as it works fine on the old client, but not on the new one?

**Comment by GBirch33** [comment]
@Kaze-No-Kai Yes it probably is. Do we have the ability to contact the artist and ask to remake using back face culling only? The reason is that back-face culling is all we support due to performance cost of state change and therefore affects on batching. Basically clone the double sided mesh and invert the normals, then it should be right.

**Comment by Kaze-No-Kai** [comment]
@GBirch33 when you have a moment could you please check the comment below. Many thanks in advance

**Comment by anicalbano** [comment]
This issue is fixed on latest (v0.54.0). The issue with the wing and the glasses is listed in other ticket https://github.com/decentraland/unity-explorer/issues/2591

https://github.com/user-attachments/assets/1e304173-6bf3-4b75-b8d8-53105d62efc8
